frontend/src/pages/Login.tsx

```diff
@@ -1,9 +1,71 @@
-import React from 'react'
+"use client";
+import React, { useState } from "react";
+import axios from "axios";
```

Comment:
What's the benefit from using `axios` over the preinstalled `fetch`? If we continue with `axios`, make sure to add it to the `package.json` file via `npm install` as I can't run it currently.